remote_development/Tiltfile

```diff
@@ -0,0 +1,82 @@
+def get_container(decoded_yaml, container_selector):
+  containers=decoded_yaml["spec"]["template"]["spec"]["containers"]
+  found_containers=[]
+
+  for container in containers:
+    if container["name"] == container_selector:
+      return container
+    
+    fail("failed to find container %s in the manifest %s" % (container_selector, containers))
+
+def get_registry(container):
+  return container["image"].rsplit(":", 1)[0]
+
+def remove_probe(container, probe):
+  if container.get(probe):
+    container.pop(probe)  
+
+def expose_ports(container, port_forwards):
```

Comment:
i don't think you need this?

From the docs: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#container-v1-core

> List of ports to expose from the container. Exposing a port here gives the system additional information about the network connections a container uses, but **is primarily informational.** [emphasis added]

Reply:
Good to know